roles/node/templates/testnet/config.toml.j2

```diff
@@ -0,0 +1,125 @@
+wasm_dir = "{{ node_wasm_dir }}"
+
+[ledger]
+genesis_time = "2024-12-05T23:07:03.000000000+00:00"
+chain_id = "{{ namada[type].chain_id }}"
+
+[ledger.shell]
+base_dir = "{{ node_network_dir }}"
+storage_read_past_height_limit = 3600
+db_dir = "{{ node_data_dir }}"
+cometbft_dir = "{{ node_cometbft_dir }}"
+tendermint_mode = "{{ 'Full' if type == 'fullnode' else 'Validator' }}"
+
+[ledger.cometbft]
+proxy_app = "{{ node_type_settings.proxy_app.address.default }}:{{ namada[type].ports.proxy_port }}"
+moniker = "{{ org }}"
+fast_sync = true
+db_backend = "goleveldb"
+db_dir = "{{ node_cometbft_data_dir }}"
+log_level = "info"
+log_format = "plain"
+genesis_file = "{{ node_cometbft_config_dir }}/genesis.json"
+priv_validator_key_file = "{{ node_cometbft_config_dir }}/priv_validator_key.json"
+priv_validator_state_file = "{{ node_cometbft_data_dir }}/priv_validator_state.json"
+priv_validator_laddr = ""
+node_key_file = "{{ node_cometbft_config_dir }}/node_key.json"
+abci = "socket"
+filter_peers = false
+
+[ledger.cometbft.rpc]
+laddr = "{{ node_type_settings.rpc.address[type] }}:{{ namada[type].ports.rpc_port }}"
+cors_allowed_origins = []
+cors_allowed_methods = ["HEAD", "GET", "POST"]
+cors_allowed_headers = ["Origin", "Accept", "Content-Type", "X-Requested-With", "X-Server-Time"]
+grpc_laddr = ""
+grpc_max_open_connections = 900
+unsafe = false
+max_open_connections = 900
+max_subscription_clients = 100
+max_subscriptions_per_client = 5
+timeout_broadcast_tx_commit = "10000ms"
+max_body_bytes = 1000000
+max_header_bytes = 1048576
+tls_cert_file = ""
+tls_key_file = ""
+pprof_laddr = ""
+
+[ledger.cometbft.p2p]
+laddr = "{{ node_type_settings.p2p.address.default }}:{{ namada[type].ports.p2p_port }}"
+external_address = ""
```

Comment:
lets add this too, please check the cometbft seeting in cosmos ansible config file, and replicate all the variable here as well